TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -133,7 +133,7 @@ class MemoryTemplateBinnedCM{
     if (nentry_ibx < getNEntryPerBin()-1) { // Max 15 stubs in each memory due to 4 bit nentries
       // write address for slot: getNEntryPerBin() * slot + nentry_ibx
 
-      writememloop:for (unsigned int icopy=0;icopy<NCOPY;icopy++) {
+      writememloop:for (unsigned int icopy=0;icopy< (signed) NCOPY;icopy++) {
```

Comment:
This PR is not reverting to this code to the old version. The old version https://github.com/cms-L1TK/firmware-hls/blob/cacc74b170cebaa0e9ba48366e96ca007dfba9d8/TrackletAlgorithm/MemoryTemplateBinnedCM.h#L117 declared icopy to be of type "signed int", not "unsigned int", which makes more sense given that NCOPY is now signed int.

Reply:
If NCOPY really can be zero, then the < test will cause an unwanted loop iteration in that case, n'est-ce pas?

Reply:
Sorry, the heat has scrambled my brain...